packages/world/src/WorldConsumer.sol

```diff
@@ -1,11 +1,12 @@
 // SPDX-License-Identifier: MIT
```

Comment:
I think Consumer is clearer than Context (which makes me think it's something World inherits, not systems), and it matches StoreConsumer

Reply:
Context was leaning on [ERC2771Context](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/metatx/ERC2771Context.sol) where this pattern originates

Reply:
I feel like Context still makes sense for this?

Reply:
yeah i see how `Consumer` would also make sense but I'd prefer to keep `Context` to draw the analogy to `ERC2771Context`